js/app/DoCommand.html

```diff
@@ -309,70 +249,24 @@
       break;
     }
     case 'studentNote': {
-      const radio = app.modal.table.querySelectorAll('input[type="radio"]');
-      let value;
-      for (let i = 0; i < radio.length; i++) {
-        if (radio[i].checked) {
-          if (radio[i].value == 'other') {
-            value = app.modal.getText();
-            break;
-          } else {
-            value = radio[i].value;
-            break;
-          }
-        }
-      }
-      if (! value) {
-        value = '';
-      }
-      if (value == 'left'){
-        let students = (
-          JSON.parse(app.pages.form.elements.form.students.value) || []
-        );
-        let studentOnForm = students.find(s => s.id == app.modal.tmp.id);
-        studentOnForm.left = true;
-        const studentChange = {
+      const value = utility.DOM.getRadioValue();
+      const students = (
+        JSON.parse(app.pages.form.elements.form.students.value) || []
+      );
+      const student = students.find(s => s.id == app.modal.tmp.id);
+      if (value == 'left') {
+        student.left = true;
+        app.pages.form.handleChange({
           target: {
-            name: 'students',
-            value: studentOnForm.name + ' left without checking out',
+            name: "students",
```

Comment:
no issue here, I'm just curious why you changed it to double quotes

Reply:
habit.   I have no preference in this project for single vs. double.  I think I started this project favoring singles, and I've since moved to favoring doubles.